opencti-platform/opencti-front/package.json

```diff
@@ -174,18 +174,20 @@
     "verify-translation": "node script/verify-translation.js",
     "sort-translation": "node script/sort-translation-files.js",
     "auto-translation:all": "npm run auto-translation:fr-front $SUBSCRIPTION_KEY && npm run auto-translation:ko-front $SUBSCRIPTION_KEY && npm run auto-translation:cn-front $SUBSCRIPTION_KEY && npm run auto-translation:ja-front $SUBSCRIPTION_KEY && npm run auto-translation:es-front $SUBSCRIPTION_KEY && npm run auto-translation:de-front $SUBSCRIPTION_KEY && npm run auto-translation:fr-back $SUBSCRIPTION_KEY && npm run auto-translation:ko-back $SUBSCRIPTION_KEY && npm run auto-translation:cn-back $SUBSCRIPTION_KEY && npm run auto-translation:ja-back $SUBSCRIPTION_KEY && npm run auto-translation:es-back $SUBSCRIPTION_KEY && npm run auto-translation:de-back $SUBSCRIPTION_KEY && npm run sort-translation",
```

Comment:
Thank you for your contribution!
italian is missing in `auto-translation:all` script.
Could you please add it and then run the script to add missing translations?